web/frontend/src/api/navigation.ts

```diff
@@ -33,44 +38,55 @@ export const setWaypoint = (client: Client, lat: number, lng: number, name: stri
   rcLogConditionally(request);
 
   return new Promise((resolve, reject) => {
-    client.navigationService.addWaypoint(request, new grpc.Metadata(), (error, response) =>
+    get(client).navigationService.addWaypoint(request, new grpc.Metadata(), (error, response) =>
       (error ? reject(error) : resolve(response)));
   });
 };
 
-export const getWaypoints = (client: Client, name: string) => {
+const transformWaypoints = (list: navigationApi.Waypoint[]) => {
```

Comment:
[nit] can the name of this function be changed? semantically (to me) "transform" implies some kind of math done on waypoints.

Reply:
Yeah that's fair since there's a lot of math in this space. Updated!